ls1intum/Apollon
# Fix SVG and PDF Export

### Checklist

### Motivation and Context


Currently, the svg and pdf exports do not render the diagrams 100% correctly.

### Description


### Steps for Testing

1. Create a diagram or multiple if you want to and export it as svg and pdf. Make sure all lines that should be visible are not cut off, e.g the borders of class diagrams.

### Test Coverage


No tests yet.
| File | Branch | Line |
|------|-------:|-----:|

### Screenshots


Before diagram created with apollon standalone pdf:
![image](https://github.com/ls1intum/Apollon/assets/84102468/2c6b8ceb-baa0-4258-a4e9-e85e72189a2e)
Now:
![image](https://github.com/ls1intum/Apollon/assets/84102468/8df3b30f-9c3a-4846-b840-d2877ce2c920)

Before: diagram created on artemis using apollon:
![image](https://github.com/ls1intum/Apollon/assets/84102468/598ed332-51b2-4641-b35d-5898bc8f52ca)

After:
![image](https://github.com/ls1intum/Apollon/assets/84102468/b51776d5-c3f9-44e9-92b8-d1d51edb210e)

More complex example:
![image](https://github.com/ls1intum/Apollon/assets/84102468/26105aef-b138-4878-b4fc-84282e530f01)
